src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/CSYamlModelGenerator.cs

```diff
@@ -1127,6 +1127,12 @@ public static ExpressionSyntax GetLiteralExpressionCore(object value, ITypeSymbo
                         SyntaxKind.NumericLiteralExpression,
                         SyntaxFactory.Literal((float)value));
                 case SpecialType.System_Double:
+                    if (value is int @int)
```

Comment:
Will `value` be a `float` if the default value is `0f`? If so consider use `Convert.ToDouble`